src/hivpy/prep.py

```diff
@@ -108,6 +130,38 @@ def get_suspect_risk_pop(self, pop: Population):
                                    COND(col.LTP_HIV_STATUS, op.eq, True),
                                    COND(col.R_PREP, op.lt, self.prob_suspect_risk_prep)))
 
+    def get_presumed_hiv_neg_pop(self, pop: Population):
+        """
+        Return the sub-population that has been tested and is HIV positive but
+        received a false negative result.
+        """
+        false_neg_pop = pop.get_sub_pop(AND(COND(col.EVER_TESTED, op.eq, True),
+                                            COND(col.HIV_STATUS, op.eq, True)))
```

Comment:
Isn't this only a false negative if `HIV_DIAGNOSED` is equal to false as well?

Reply:
You're right, that condition should be included here.